config/routes.rb

```diff
@@ -30,6 +25,13 @@
     delete 'logga_ut' => 'devise/sessions#destroy', as: :destroy_user_session
   end
 
+  # tags and exlinks
+  get 'ex_links/tags' => 'ex_links#list_tags'
+  get 'ex_links/del_unused_tags' => 'ex_links#del_unused_tags'
+  get 'ex_links/check_dead' => 'ex_links#check_dead'
+  get 'ex_links/check_expired' => 'ex_links#check_expired'
+  resources :ex_links
```

Comment:
I don't need any response from this actions, they basically run an appropriate routine and then refresh the page. They do not work with specific object, rather iterate through all and do some action (either delete them in case of tags or mark them inactive in case of links).